nextcord/types/appinfo.py

```diff
@@ -35,7 +35,6 @@ class BaseAppInfo(TypedDict):
     name: str
     verify_key: str
     icon: Optional[str]
-    summary: str
```

Comment:
I think you are the first one to edit this file, copyright must be changed corresponding to other files aswell (please also check this in the other files)

Reply:
ah ic.

Reply:
I am actually not sure about this one @TAG-Epic

Reply:
Yeah feel free to add it